yarp/ReadOnlyControlBoardRemapper/ReadOnlyRemoteControlBoardRemapper.h

```diff
@@ -0,0 +1,142 @@
+/*
+* Copyright (C) 2016 iCub Facility - Istituto Italiano di Tecnologia
+* Author: Lorenzo Natale, Silvio Traversaro, Francesco Romano
+* CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+*/
+
+#ifndef YARP_DEV_READONLYCONTROLBOARDREMAPPER_READONLYREMOTECONTROLBOARDREMAPPER_H
+#define YARP_DEV_READONLYCONTROLBOARDREMAPPER_READONLYREMOTECONTROLBOARDREMAPPER_H
+
+#include <yarp/dev/PolyDriver.h>
+
+#include "ReadOnlyControlBoardRemapper.h"
+
+
+namespace yarp {
+namespace dev {
+    class ReadOnlyRemoteControlBoardRemapper;
+}
+}
+
+/**
+ *  @ingroup dev_impl_wrapper
+ *
+ * \section RemoteControlBoardRemapper
+ * A device that takes a list of axes from multiple controlboards, a list
+ * of remote controlboards in which this axes are located, that is opening
+ * all the remote controlboards but is exposing them
+ *
+ *
+ *  Parameters required by this device are:
```

Comment:
It has been copy-pasted from the original source. Still to be modified with the new parameters